concepts/let/about.md

```diff
@@ -0,0 +1,85 @@
+# About
+
+Elm is a fully functional language, so there are no statements, just expressions, and functions are always a single expression.
+
+This encourages small, simple functions and is generally a good thing, but sometimes functions are unavoidably more complex, and we need a way to ensure that these functions can be written elegantly.
+
+This is where `let` expressions come in. They have two parts, a `let` section where you can create as many definitions as you need, and then an `in` section, which is a single expression that can use these definitions.
+
+In computing terms, `let` creates a scope that is local to the expression in `in`.
+
+The let expression is useful in a few situations:
+
+- For simplifying expressions
+- For naming the results of intermediate expressions
+- Splitting an expression in to smaller bits if it is getting large
+
+You can do most things inside a `let` expression, including destructuring (more details on this in later exercises), however the following are not allowed:
```

Comment:
The contents of this document are presented "stand-alone," that is without the context of an exercise. As such, the concept documents should not reference any exercise.

Reply:
Good point, I've pushed a commit